common/include/pcl/impl/point_types.hpp

```diff
@@ -506,20 +506,22 @@ namespace pcl
   PCL_EXPORTS std::ostream& operator << (std::ostream& os, const PointXYZRGBA& p);
   /** \brief A point structure representing Euclidean xyz coordinates, and the RGBA color.
     *
-    * The RGBA information is available either as separate r, g, b, or as a
-    * packed std::uint32_t rgba value. To pack it, use:
+    * The RGBA information is available either as separate r, g, b and a uint8_t values,
+    * or as a packed std::uint32_t rgba value. To pack it, use:
     *
     * \code
-    * int rgb = ((int)r) << 16 | ((int)g) << 8 | ((int)b);
+    * std::uint32_t rgba = (a << 24 | (r << 16 | (g << 8) | b;
```

Comment:
Assuming `a`, `r`, `g`, and `b` are `std::uint8_t`, they have to be cast to `std::uint32_t` before shifting.